src/components/DayTime.tsx

```diff
@@ -86,9 +86,9 @@ const Span = styled.span`
     color: ${dayTimeColor};
     font-weight: bold;
     text-align: left;
-    padding-left: 4%;
+    padding-left: 3%;
 `;
-// padding-right: 1em;
-// @media only screen and (min-width: 768px) {
-//     padding-right: 2.5em;
-// }
\ No newline at end of file
+
+const Label= styled.label`
```

Comment:
I'd expect your editor to autoformat the code upon save/commit but otherwise you'll need to manually format it. Put a space before the `=` sign.